mutiny-core/src/nostr/mod.rs

```diff
@@ -122,7 +122,7 @@ impl<S: MutinyStorage> NostrManager<S> {
         relays.push("wss://relay.primal.net".to_string());
         relays.push("wss://relay.damus.io".to_string());
 
-        // add blastr for default sending
+        // add strfry for default sending
         relays.push("wss://relay.mutinywallet.com".to_string());
```

Comment:
tbh we should add blastr back, can you do that

Reply:
you mean to have both relays? yes, can do

Reply:
you mean to have both relays? yes, can do